libgraph/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ set(sources
         src/Properties.cpp
         src/PropertyGraph.cpp
         src/PropertyGraphRetractor.cpp
-        src/PropertyIndex.cpp
+    src/EntityIndex.cpp
```

Comment:
nit: clang-tidy didn't catch this spacing issue?

Reply:
Ooops. There is no format checking for cmake code. :-) I'll fix it.

Can I get a review on https://github.com/KatanaGraph/katana-enterprise/pull/3026 as well. It's just updating the users in enterprise.